Build instructions from quickstart do not work

### Describe the bug

Continuation of #1236. I cannot re-open the issue, but I am facing the same exact issue.

The `eclipselink` getting-started instructs users to run the following command:

```
./gradlew clean :polaris-quarkus-server:assemble :polaris-quarkus-admin:assemble \                                                                                                                     
   -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4 \
   -Dquarkus.container-image.tag=postgres-latest \
   -Dquarkus.container-image.build=true \
   --no-build-cache
```

But after running, the command fails with the following error:

```
Execution failed for task ':polaris-quarkus-admin:quarkusAppPartsBuild'.
> There was a failure while executing work items
   > A failure occurred while executing io.quarkus.gradle.tasks.worker.BuildWorker
      > io.quarkus.builder.BuildException: Build failure: Build failed due to errors
                [error]: Build step io.quarkus.container.image.docker.deployment.DockerProcessor#dockerBuildFromJar threw an exception: java.lang.RuntimeException: Execution of 'docker build -f /Users/ahemani/Development/polaris/quarkus/admin/src/main/docker/Dockerfile.jvm -t docker.io/apache/polaris-admin-tool:postgres-latest /Users/ahemani/Development/polaris/quarkus/admin' failed. See docker output for more details
                at io.quarkus.container.image.docker.common.deployment.CommonProcessor.containerRuntimeException(CommonProcessor.java:261)
                at io.quarkus.container.image.docker.common.deployment.CommonProcessor.buildImage(CommonProcessor.java:249)
                at io.quarkus.container.image.docker.deployment.DockerProcessor.createContainerImage(DockerProcessor.java:125)
                at io.quarkus.container.image.docker.deployment.DockerProcessor.createContainerImage(DockerProcessor.java:31)
                at io.quarkus.container.image.docker.common.deployment.CommonProcessor.buildFromJar(CommonProcessor.java:87)
                at io.quarkus.container.image.docker.deployment.DockerProcessor.dockerBuildFromJar(DockerProcessor.java:62)
                at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
                at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:856)
                at io.quarkus.builder.BuildContext.run(BuildContext.java:255)
                at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
                at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2675)
                at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2654)
                at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1627)
                at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1594)
                at java.base/java.lang.Thread.run(Thread.java:1583)
                at org.jboss.threads.JBossThread.run(JBossThread.java:499)
```

Upon running the Docker command that was failing, we see that there is a `build` directory that is expected to exist in the image - but it does not exist:

```
ahemani@F7Q5C90CJX polaris % docker build -f /Users/ahemani/Development/polaris/quarkus/admin/src/main/docker/Dockerfile.jvm -t docker.io/apache/polaris-admin-tool:postgres-latest /Users/ahemani/Development/polaris/quarkus/admin
[+] Building 0.5s (8/8) FINISHED                                                                                                                                                                                             docker:desktop-linux
 => [internal] load build definition from Dockerfile.jvm                                                                                                                                                                                     0.0s
 => => transferring dockerfile: 1.66kB                                                                                                                                                                                                       0.0s
 => [internal] load metadata for registry.access.redhat.com/ubi9/openjdk-21-runtime:1.21-1.1741781258                                                                                                                                        0.4s
 => [internal] load .dockerignore                                                                                                                                                                                                            0.0s
 => => transferring context: 2B                                                                                                                                                                                                              0.0s
 => [1/4] FROM registry.access.redhat.com/ubi9/openjdk-21-runtime:1.21-1.1741781258@sha256:360822c35c5741f542ab78fe123e6c4d9b68e0113a88d6e0250bb1f377b17f29                                                                                  0.0s
 => => resolve registry.access.redhat.com/ubi9/openjdk-21-runtime:1.21-1.1741781258@sha256:360822c35c5741f542ab78fe123e6c4d9b68e0113a88d6e0250bb1f377b17f29                                                                                  0.0s
 => [internal] load build context                                                                                                                                                                                                            0.0s
 => => transferring context: 2B                                                                                                                                                                                                              0.0s
 => CACHED [2/4] RUN groupadd --gid 10001 polaris       && useradd --uid 10000 --gid polaris polaris       && chown -R polaris:polaris /opt/jboss/container       && chown -R polaris:polaris /deployments                                   0.0s
 => CACHED [3/4] WORKDIR /deployments                                                                                                                                                                                                        0.0s
 => ERROR [4/4] COPY --chown=polaris:polaris build/*-runner.jar /deployments/polaris-server-admin-tool.jar                                                                                                                                   0.0s
------
 > [4/4] COPY --chown=polaris:polaris build/*-runner.jar /deployments/polaris-server-admin-tool.jar:
------
Dockerfile.jvm:40
--------------------
  38 |     ENV PWD=/deployments
  39 |     
  40 | >>> COPY --chown=polaris:polaris build/*-runner.jar /deployments/polaris-server-admin-tool.jar
  41 |     
  42 |     ENTRYPOINT [ "java", "-jar", "/deployments/polaris-server-admin-tool.jar" ]
--------------------
ERROR: failed to solve: lstat /build: no such file or directory
```

### To Reproduce

_No response_

### Actual Behavior

_No response_

### Expected Behavior

_No response_

### Additional context

_No response_

### System information

_No response_

Also attempted to point `polaris-server-admin-tool.jar` towards the `quarkus-run.jar` file in case it there was a typo - but got the following error:

```
ahemani@F7Q5C90CJX polaris % docker run --rm -it apache/polaris-admin-tool:1.0.0-incubating-SNAPSHOT
Error: Could not find or load main class io.quarkus.bootstrap.runner.QuarkusEntryPoint
Caused by: java.lang.ClassNotFoundException: io.quarkus.bootstrap.runner.QuarkusEntryPoint
```

Per this, I'm assuming my original guess that this JAR isn't the correct one I'm looking for is right.

@snazy, as committer of #1082 - if you have any tips, it would be greatly appreciated. Here's where I'm at thus far:
* I've tried to swap the now-nonexistent `polaris-quarkus-admin-*-runner.jar` (due to changes in #1082) with the `quarkus-run.jar`, which per my understanding is the new runner jar for the `admin` module (as seen by [L39 in `run.sh`](https://github.com/apache/polaris/pull/1082/files#diff-8ecbdc4cc60c3f26f9f2f88d6e1943c37554f8647ae5e07a13961f3f989f6280R39)
* However, I'm still getting the same error as a few comments above:
```
ahemani@F7Q5C90CJX polaris % docker run --rm -it apache/polaris-admin-tool:1.0.0-incubating-SNAPSHOT       
INFO exec -a "java" java -XX:MaxRAMPercentage=80.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+ExitOnOutOfMemoryError -cp "." -jar /deployments/quarkus-run.jar 
INFO running in /deployments
Error: Could not find or load main class io.quarkus.bootstrap.runner.QuarkusEntryPoint
Caused by: java.lang.ClassNotFoundException: io.quarkus.bootstrap.runner.QuarkusEntryPoint
```

^This is after me attempting to change the Docker image by making the following changes in the admin Dockerfile (mirrored from the `server` Dockerfile):
```
COPY --chown=polaris:polaris build/quarkus-app/*.jar /deployments/quarkus-run.jar
COPY --chown=polaris:polaris build/quarkus-app/app/ /deployments/app/
COPY --chown=polaris:polaris build/quarkus-app/quarkus/ /deployments/quarkus/

ENTRYPOINT [ "java", "-jar", "/deployments/quarkus-run.jar" ]
#ENV JAVA_APP_JAR="/deployments/quarkus-run.jar"
```

I understand I'm not using the `run.sh` file for running `quarkus-run.jar`, but looking at the shell code itself, it does not seem to be doing anything complex on its own. Running using `java` CLI does not seem to be doing anything too different. Is there anything that stands out?

----

This is blocking the Getting Started module with EclipseLink, as the Docker image for that Getting Started requires both the server and admin modules to be able to run. I'm hoping to use this Docker image to enhance our Getting Started experience on all Cloud Providers - so anything that might help unblock me would be greatly appreciated!

--- a/quarkus/admin/src/main/docker/Dockerfile.jvm
+++ b/quarkus/admin/src/main/docker/Dockerfile.jvm
@@ -31,12 +31,15 @@ RUN groupadd --gid 10001 polaris \
       && chown -R polaris:polaris /deployments
 
 USER polaris
-WORKDIR /deployments
+WORKDIR /home/polaris
 ENV USER=polaris
 ENV UID=10000
 ENV HOME=/home/polaris
-ENV PWD=/deployments
 
-COPY --chown=polaris:polaris build/*-runner.jar /deployments/polaris-server-admin-tool.jar
+# We make four distinct layers so if there are application changes the library layers can be re-used
+COPY --chown=polaris:polaris build/quarkus-app/lib/ /deployments/lib/
+COPY --chown=polaris:polaris build/quarkus-app/quarkus-run.jar /deployments/polaris-admin-tool.jar
+COPY --chown=polaris:polaris build/quarkus-app/app/ /deployments/app/
+COPY --chown=polaris:polaris build/quarkus-app/quarkus/ /deployments/quarkus/
 
-ENTRYPOINT [ "java", "-jar", "/deployments/polaris-server-admin-tool.jar" ]
+ENTRYPOINT [ "java", "-jar", "/deployments/polaris-admin-tool.jar" ]

--- a/quarkus/admin/src/main/java/org/apache/polaris/admintool/PolarisAdminTool.java
+++ b/quarkus/admin/src/main/java/org/apache/polaris/admintool/PolarisAdminTool.java
@@ -26,7 +26,7 @@ import picocli.CommandLine.HelpCommand;
 
 @TopCommand
 @Command(
-    name = "polaris-quarkus-admin-runner.jar",
+    name = "polaris-admin-tool.jar",
     mixinStandardHelpOptions = true,
     versionProvider = PolarisVersionProvider.class,
     description = "Polaris Admin Tool",

